### PR TITLE
[CBRD-20037]  Write the timezone checksum in all the available databases on disk

### PR DESCRIPTION
**Pull request by asturzu**

### CODE REVIEW

File: src/executables/util_sa.c [hunk 1]
```diff
@@ -3933,49 +3929,86 @@ gen_tz (UTIL_FUNCTION_ARG * arg)
...
+	  if (dir == NULL)
+	    {
+	      er_set (ER_ERROR_SEVERITY, ARG_FILE_LINE, ER_OUT_OF_VIRTUAL_MEMORY, 1, sizeof (DB_INFO));
+	      exit_status = EXIT_FAILURE;
+	      goto exit;
+	    }
+	  dir->name = (char *) calloc (strlen (db_name) + 1, sizeof (char));
+	  if (dir->name == NULL)
+	    {
+	      er_set (ER_ERROR_SEVERITY, ARG_FILE_LINE, ER_OUT_OF_VIRTUAL_MEMORY, 1, sizeof (DB_INFO));
```
Comment (eseokoh) [inline src/executables/util_sa.c]: Please correct `sizeof (DB_INFO)`. It must be `strlen (db_name) + 1`.

Comment (asturzu) [inline src/executables/util_sa.c]: Dear Mr Oh,

The dir variable is of DB_INFO type so the allocation with sizeof(DB_INFO) is correct, strlen(db_name) + 1 is used when I allocate memory for the db_name variable.

                                                   Best regards,
                                                             Antonio

Comment (eseokoh) [inline src/executables/util_sa.c]: @asturzu 
I'm sorry. I was not clear.
I wanted to point the line 3963, er_set. 
Thank you.

Comment (asturzu) [inline src/executables/util_sa.c]: @eseokoh ,

I updated the patch with the modification you requested.

                   Antonio

**Comment by eseokoh** [comment]
Thanks for the update. 
Would you please merge your change? :smiley:

**Pull request merged**